frontend/app/utils/date-utils.ts

```diff
@@ -73,7 +73,28 @@ export function isValidTimeZone(timeZone: string): boolean {
 }
 
 /**
-<<<<<<< HEAD
+ * @param timezone - The IANA time zone name (e.g., 'America/New_York', 'Europe/London').
+ * @param date - Optional date or timestamp to use. If not provided, the current date and time are used.
+ *               Can be a number (milliseconds since epoch), a string (parsable by `new Date()`), or a Date object.
+ * @returns The date formatted in both English ("MMM dd, yyyy") and French ("dd MMM. yyyy").
+ */
+export function dateToLocalizedText(timezone?: string, date?: number | string | Date): { dateEn: string; dateFr: string } {
```

Comment:
I don't think the timezone should be optional here.

I feel like allowing the timezone to be optional creates potential for bugs... where a calling function omits the timezone (either accidentally or in error), which will cause the returned string to be in UTC.